### PR TITLE
[mypy] ignore 5

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10508)

## Description
ignore mypy errors in

1. AlphaVantage
2. FindEmailCampaign
3. DateTimeToADTime
4. ComputerVisionEngine
5. DNSDB_v2
6. Rapid7ThreatCommand
7. MicrosoftTeams
8. Rapid7AppSec
9. Rapid7_Nexpose
10. SplunkPy
11. TrendMicroVisionOneV3
12. VMware
13. Zendeskv2
14. CortexXDRIRCTF

### CODE REVIEW

File: Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
 
-import datetime  # type: ignore[no-redef]
+import datetime  # type: ignore[assignment]
```
[the remark this answers is not in the document]
Comment (shmuel44) [reply Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py]: fixed `from datetime import datetime`

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 import json
 import urllib3
-from datetime import datetime, timedelta, UTC
+from datetime import datetime, timedelta, UTC  # type: ignore[attr-defined]
```
Comment (shmuel44) [inline Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py:10]:
```suggestion
from datetime import datetime, timedelta, UTC
```

File: Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py [hunk 3]
```diff
@@ -602,7 +602,7 @@ def parse_rate_limit_int(i):
 
 def parse_unix_time(ts) -> str:
     try:
-        return datetime.datetime.utcfromtimestamp(ts).strftime("%Y-%m-%dT%H:%M:%SZ")  # type: ignore[attr-defined]
+        return datetime.utcfromtimestamp(ts).strftime("%Y-%m-%dT%H:%M:%SZ")  # type: ignore[attr-defined]
```
Comment (dorschw) [inline Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py:605]: can we remove the ignore here? or, why is there an undefined attribute?

**Review by dorschw — APPROVED** [review]

**Review by kobymeir — APPROVED** [review]

**Comment by shmuel44** [comment]
@kobymeir needs force merge 

- The UT coverage has decreased by 0.5%.
- DO106
- RM106 - An invalid keyword 'demisto' was found.
- RN106 - Release notes were not found.
- PA114 - The pack version needs to be increased.
- The nexpose_test tpb failed, but I only added # type: ignore[override].

**Pull request merged**